src/encryption/encryption.go

```diff
@@ -107,16 +106,3 @@ func LogGroups(groups []Group, logger *zap.SugaredLogger) {
 func DoEncrypt(projectState *state.State, unencryptedGroups []Group) {
 
 }
-
-func updateMapPart(mapPart interface{}, keyPath path, value interface{}) interface{} {
-	if len(keyPath) == 1 {
-		return keyPath[0].Update(mapPart, fmt.Sprint(value))
-	}
-	newPart := updateMapPart(keyPath[0].Get(mapPart), keyPath[1:], value)
-	return keyPath[0].Update(mapPart, newPart)
-}
-
-func updateObjectContent(content *orderedmap.OrderedMap, keyPath path, value interface{}) *orderedmap.OrderedMap {
-	newContent := updateMapPart(*content, keyPath, value).(orderedmap.OrderedMap)
-	return &newContent
-}
```

Comment:
stary nacrt implementacie update funkcie. Povodne som to mal cez metody step interface ale vypadalo to divne tak som to prepisal vid nizsie.